packages/components/src/slot-fill/bubbles-virtually/fill.tsx

```diff
@@ -15,26 +14,9 @@ import {
  */
 import SlotFillContext from './slot-fill-context';
 import StyleProvider from '../../style-provider';
+import useForceUpdate from '../use-force-update';
 import type { FillComponentProps } from '../types';
 
-function useForceUpdate() {
-	const [ , setState ] = useState( {} );
-	const mountedRef = useRef( true );
-
-	useEffect( () => {
-		mountedRef.current = true;
-		return () => {
-			mountedRef.current = false;
-		};
-	}, [] );
-
-	return () => {
-		if ( mountedRef.current ) {
-			setState( {} );
-		}
-	};
-}
```

Comment:
This is extracted to a separate module because the `useForceUpdate` hook is used by two components now. Also with React 18 we no longer need `mountedRef` because calling `setState` after unmount is no longer a problem.